<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ToolTab</class>
 <widget class="QWidget" name="ToolTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>224</width>
    <height>409</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="toolToolsLayout"/>
   </item>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="lineWidth">
      <number>1</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="handleWidth">
      <number>3</number>
     </property>
     <widget class="QTreeWidget" name="toolParamTable">
      <column>
       <property name="text">
        <string notr="true">1</string>
       </property>
      </column>
     </widget>
     <widget class="QTextBrowser" name="toolTextBox">
      <property name="font">
       <font>
        <family>Courier</family>
        <pointsize>8</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="cursor" stdset="0">
       <cursorShape>IBeamCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(217, 255, 255);
font: 8pt &quot;Courier&quot;;
;</string>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Courier'; font-size:8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
  <action name="actionActive">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="Tool.qrc">
     <normaloff>:/Icon8/Images/new_icons/icons8-toggle-on.png</normaloff>:/Icon8/Images/new_icons/icons8-toggle-on.png</iconset>
   </property>
   <property name="text">
    <string>Active</string>
   </property>
   <property name="toolTip">
    <string>Active Tool?</string>
   </property>
  </action>
  <action name="actionApplyToTheory">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="Tool.qrc">
     <normaloff>:/Icon8/Images/new_icons/icons8-einstein-yes.png</normaloff>:/Icon8/Images/new_icons/icons8-einstein-yes.png</iconset>
   </property>
   <property name="text">
    <string>Apply to Theory</string>
   </property>
   <property name="toolTip">
    <string>Apply Tool to Theory</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="Tool.qrc"/>
 </resources>
 <connections/>
</ui>
